torch/csrc/jit/codegen/cuda/lower_loops.cpp

```diff
@@ -24,6 +24,25 @@ LoopNestGenerator::LoopNestGenerator(
   generate(exprs);
 }
 
+namespace {
```

Comment:
nit: empty lines after namespace { and before the closing }

Reply:
Thanks. Added.